org.kde.kstars.json

```diff
@@ -98,49 +168,369 @@
+                "printf '#!/usr/bin/env python3\\nfrom SCons.Script import main\\nmain()\\n' > /app/bin/scons",
+                "chmod +x /app/bin/scons"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://files.pythonhosted.org/packages/c8/c1/30176c76c1ef723fab62e5cdb15d3c972427a146cb6f868748613d7b25af/scons-4.9.1.tar.gz",
+                    "sha256": "bacac880ba2e86d6a156c116e2f8f2bfa82b257046f3ac2666c85c53c615c338"
+                }
+            ]
+        },
+        {
+            "name": "gpsd",
+            "buildsystem": "simple",
+            "builddir": true,
+            "build-commands": [
+                "scons prefix=/app",
+                "find . -name 'libgps.so' -exec install -Dm755 {} /app/lib/libgps.so \\;",
+                "install -Dm644 include/gps.h /app/include/gps.h",
+                "install -Dm644 include/libgps.h /app/include/libgps.h",
+                "install -Dm644 include/libgpsmm.h /app/include/libgpsmm.h"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.savannah.gnu.org/releases/gpsd/gpsd-3.26.1.tar.gz",
+                    "sha256": "dc7e465968c1540e61bc57c7586d6a57a0047212a014efdad348f907bc2e0990",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 6846,
+                        "stable-only": true,
+                        "url-template": "hhttps://download.savannah.gnu.org/releases/gpsd/gpsd-$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "zeromq",
+            "buildsystem": "autotools",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/zeromq/libzmq/releases/download/v4.3.5/zeromq-4.3.5.tar.gz",
+                    "sha256": "6653ef5910f17954861fe72332e68b03ca6e4d9c7160eb3a8de5a5a913bfab43",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "project-id": 16245,
+                        "stable-only": true,
+                        "url-template": "https://github.com/zeromq/libzmq/releases/download/$version/zeromq-$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "qt6-qtdatavis3d",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://invent.kde.org/qt/qt/qtdatavis3d/-/archive/v6.9.0/qtdatavis3d-v6.9.0.tar.gz",
+                    "sha256": "b504d5da10832eb6dfae85956c4ae0d3aa77d99d3be471e001cfcb8c828809a5"
+                }
+            ]
+        },
+        {
+            "name": "qt6-qtcharts",
```

Comment:
available in [runtime](https://invent.kde.org/packaging/flatpak-kde-runtime/-/blob/qt6.9/org.kde.Sdk.json.in#L555)